src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static nature() {
+    this.alive = this.alive.filter((dead) =>
+      dead.health !== 0);
+  }
+  constructor(name) {
+    this.health = 100;
+    this.name = name;
+
+    Animal.alive.push(this);
+  }
+
+  injure(healthPoints) {
+    this.health -= healthPoints;
+
+    if (this.health <= 0) {
+      Animal.alive.filter((animal) => animal.health <= 0);
```

Comment:
[CODE PERFORMANCE] Avoid using nested loops for removing dead animals from the 'alive' array. The 'filter' method is called inside 'injure', but it does not update 'Animal.alive'. Instead, use 'Animal.alive = Animal.alive.filter...' to directly assign the filtered array.